### DISCUSSION TITLE
the workbook's Unprotect function do not work

### DISCUSSION THREAD

**Discussion by fightingpjh** [mhammond/pywin32#2547] [General]
According to :   https://docs.microsoft.com/zh-CN/office/vba/api/excel.workbook.unprotect
code: 
xcl = win32com.client.Dispatch('Excel.Application')
xcl.visible = True
pw_str = raw_input("Enter password: ")
try:
    wb = xcl.workbooks.open(filename, 0, False, None, pw_str, pw_str)
except Exception as e:
    print "Error:", str(e)
    sys.exit()
wb.Unprotect(pw_str)
wb.UnprotectSharing(pw_str)
wb.Save()
xcl.Quit()

it does't work, the excel file still protected.

**Comment by mhammond** [comment]
See https://github.com/mhammond/pywin32#support for support requests.
